lib/benchee/formatters/csv/statistics.ex

```diff
@@ -1,85 +1,106 @@
 defmodule Benchee.Formatters.CSV.Statistics do
+  @moduledoc """
+  Functionality for converting Benchee scenarios to csv with statistics.
+  """
+
   alias Benchee.{Statistics, Benchmark.Scenario}
 
-  @column_descriptors [
+  @headers_with_memory [
     "Name",
     "Input",
     "Iterations per Second",
-    "Average",
-    "Standard Deviation",
     "Standard Deviation Iterations Per Second",
-    "Standard Deviation Ratio",
-    "Median",
-    "Minimum",
-    "Maximum",
-    "Sample Size"
+    "Run Time Average",
+    "Run Time Median",
+    "Run Time Minimum",
+    "Run Time Maximum",
+    "Run Time Standard Deviation",
+    "Run Time Standard Deviation Ratio",
+    "Run Time Sample Size",
+    "Memory Usage Average",
+    "Memory Usage Median",
+    "Memory Usage Minimum",
+    "Memory Usage Maximum",
+    "Memory Usage Standard Deviation",
+    "Memory Usage Standard Deviation Ratio",
+    "Memory Usage Sample Size"
   ]
-  @moduledoc """
-    Functionality for converting Benchee scenarios to csv with statistics.
-  """
 
-  @doc """
-    Adds headers to given measurements.
+  @headers_without_memory [
+    "Name",
+    "Input",
+    "Iterations per Second",
+    "Standard Deviation Iterations Per Second",
+    "Run Time Average",
+    "Run Time Median",
+    "Run Time Minimum",
+    "Run Time Maximum",
+    "Run Time Standard Deviation",
+    "Run Time Standard Deviation Ratio",
+    "Run Time Sample Size"
+  ]
 
-    ## Examples
-      iex> Benchee.Formatters.CSV.Statistics.add_headers([])
-      [["Name", "Input", "Iterations per Second", "Average", "Standard Deviation",
-      "Standard Deviation Iterations Per Second", "Standard Deviation Ratio",
-      "Median", "Minimum", "Maximum", "Sample Size"]]
-  """
-  def add_headers(measurements) do
-    [@column_descriptors | measurements]
+  @doc false
+  def add_headers([first_row | _] = measurements) do
+    case length(first_row) do
+      11 -> [@headers_without_memory | measurements]
+      18 -> [@headers_with_memory | measurements]
```

Comment:
These numbers seem magic and it's not very resilient for the future (when we count reductions for instance) - how about just checking in the suite/configuration if memory measurements should have been run?